pyproject.toml

```diff
@@ -1,3 +1,51 @@
+[tool.poetry]
+name = "llm-app"
+version = "0.1.0"
+description = "A Pathway LLM App"
+authors = ["Your Name <you@example.com>"]
+license = "MIT License"
+readme = "README.md"
+packages = [{include = "llm_app"}]
```

Comment:
Does the poetry default not work here?

Reply:
I'd want to include the data folder as well. WDYT?